sort.php

```diff
@@ -61,4 +61,13 @@ function bubbleSort($sort_array, $checkEquality)
     $fileName = 'php://stdin';
 }
 
-echo implode("", bubbleSort(checkForNextLine(file($fileName)), $funcCallback));
\ No newline at end of file
+if (array_search("-o", $argv)) {
```

Comment:
Please check for -o the same way as for -n.

Reply:
This is not possible because after the ``-o`` option an argument is required and if you do this without an argument, the option with getopt () is not recognized at all, then it only says array (0) but if I use array_search through the $ argv the option is always recognized.

Reply:
The ``-o`` option is only recognized by ``getopt ()`` if there is an ``argument`` afterwards.

Reply:
That's legit. Why should one use -o without an argument?